templates/tools/dockerfile/php7_deps.include

```diff
@@ -29,17 +29,17 @@ RUN cd /var/local ${'\\'}
   && tar -zxvf bison-3.4.2.tar.gz ${'\\'}
   && cd /var/local/bison-3.4.2 ${'\\'}
   && ./configure ${'\\'}
-  && make ${'\\'}
+  && make -j4 ${'\\'}
```

Comment:
Are we ok with hardcoding `4` here?

Reply:
`-j4` is commonly used throughout our docker files but `$(nproc)` would be more adaptable.

Reply:
I think $(nproc) is not necessarily better since in many of our jobs, we're running multiple run_tests.py instances (where each one has their own docker container for isolation) in parallel, and having multiple `make -j$(nproc)` might be excessive. `-j4` was fine IMHO.

Reply:
@jtattermusch This docker image particularly is built manually on a local machine. I don't believe this part is being used by CI workers.